src/deployer-watch/deployer-watch.js

```diff
@@ -0,0 +1,67 @@
+const { Finding, FindingSeverity, FindingType } = require('forta-agent');
+
+const addressList = require('./deployer-watch.json');
```

Comment:
It's implied in the code that the deployer-watch.json file is a whitelist (including Deployer).  Should we rename it to whitelist.json to make that more clear?

Reply:
Wasn't sure if we wanted to stick with the convention of having handler specific code living in a json file of the same name. I agree that `whitelist.json` is more clear, so I'll go ahead and rename

Reply:
Actually, because it's a little odd to include the Deployer in the whitelist, I've went ahead and shuffled the .json file around a little more. I think it's more clear now.